### PR TITLE
css updates

### PR DESCRIPTION
**Pull request by rvennam**
@Austinauth @l2fprod

### CODE REVIEW

File: web/public/routes/home/home.css [hunk 1]
```diff
@@ -52,6 +55,8 @@
 
 .image-box {
   margin-bottom: 20px;
+  padding-bottom: 20px;
+  border-bottom: 1px solid #eaeaea;
```
Comment (l2fprod) [inline web/public/routes/home/home.css]: @rvennam 

the color definition should be moved to darkvision-light.css and its counterpart created in darkvision-dark.css. Check if you could add the "border-box" to the element. It is the light border color used in other places in the ui.

no color in darkvision.css, home.css or video.css to support the theme switching.

Comment (rvennam) [reply web/public/routes/home/home.css]: moved color to light and dark css. Not sure what you mean about the border-box.

Comment (l2fprod) [reply web/public/routes/home/home.css]: @rvennam 

"border-box" is an existing class defined in darkvision-light/dark.css

you can define image-box in home.css as:
```
border-bottom: 1px solid;
```
without the color then you can add the class "border-box" to the element that has image-box already.

And if the color of the border-box is not what you want, then fine, you add a new "image-box" class in darkvision-light/dark that has only the color definition for the border:
```
border-color: #eaeaea;
```

This way when darkvision.css, home.css and darkvision-light/dark are all put together the css classes are merged and you get your colored border definition.

Look for references to "border-box" in the html code.

darkvision.css, home.css, video.css -> all but colors
darkvision-light/dark -> only colors.

Comment (rvennam) [reply web/public/routes/home/home.css]: updated.

**Pull request merged**